ERCS/erc-7562.md

```diff
@@ -0,0 +1,393 @@
+The alt-mempool "topic" is a unique identifier. By convention, this is the IPFS hash of the document describing (in clear test and YAML file) the specifics of this alt mempool
+
+* **[ALT-010]** The bundler listens to the alt-mempool "topic" over the P2P protocol
+* **[ALT-020]** The alt mempool rules MUST be checked only when a canonical rule is violated
+    * That is, if validation follows the canonical rules above, it is not considered part of an alt-mempool.
+*  **[ALT-021]** Such a `UserOperation` (that violates the canonical rules) is checked against all the "alternate mempools", and is considered part of all those alt-mempools 
+* **[ALT-030]** Bundlers SHOULD forward `UserOperations` to other bundlers only once, regardless of how many (shared) alt-mempools they have. \
+  The receiving bundler validates the `UserOperations`, and based on the above rules (and subscribed alt-mempools) decides which alt-mempools to propagate it to.
+* **[ALT-040]** opsInclude and opsSeen of entities are kept per alt-mempool. That is, an entity can be considered throttled (or banned) in one mempool, while still active on another.
+
+### Alt-mempool Reputation
+
+Alt-mempools are served by the same bundlers participating in the canonical mempool, but change the rules and may introduce denial-of-service attack vectors. To prevent them from taking the canonical mempool or other alt mempools down with them, a reputation is managed for each. An alt mempool that causes too many invalidations gets throttled. This limits the scope of the attack and lets the bundler continue doing its work for other mempools.
+
+* **[AREP-010]** each alt-mempool manages "opsSeen" and "opsIncluded", much like entities. The opsSeen is incremented after `UserOperation` initial validation, where it is considered part of this mempool.
+    The "opsIncluded" is incremented after this UserOperation is included on-chain (either by this bundler, or another)
+* **[AREP-020]** the alt-mempool becomes THROTTLED based on the [Reputation Calculation](#reputation-calculation)
+
+## Rationale
+
+All transactions initiated by EOAs have an implicit validation phase where balance, nonce, and signature are
+checked to be valid for the current state of the Ethereum blockchain.
+Once the transaction is checked to be valid by a node, only another transaction by the same EOA can modify the Ethereum
+state in a way that makes the first transaction invalid.
+
+With Account Abstraction, however, the validation can also include an arbitrary EVM code and rely on storage as well,
+which means that unrelated `UserOperations` or transactions may invalidate each other.
+
+If not addressed, this would make the job of maintaining a mempool of valid `UserOperations` and producing valid
+bundles computationally infeasible and susceptible to DoS attacks.
+
+This document describes a set of validation rules that if applied by a bundler before accepting a `UserOperation`
+into the mempool can prevent such attacks.
```

Comment:
These paragraphs justify the proposal as a whole (eg. "this document describes..."), and so should be in the Motivation section. The Rationale section should explain technical choices made within the document.

In other words:

> Motivation: "Why do we need a shed?"
> Rationale: "Why is the shed red?"